source/projects/overview.txt

```diff
@@ -0,0 +1,50 @@
+.. _rm-project-overview:
+
+=======================
+User Interface Overview
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+When you open a project in Relational Migrator, the project user interface displays.
+You can manage many aspects of your Relational Migrator project from this interface:
+
+.. figure:: /img/projects/project-overview.png
+   :alt: Project Overview Screen
+   :lightbox:
+
+UI Elements
+-----------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - UI Element
+     - Description
+
+   * - Top Navigation Bar
+     - Switch between different contexts, including :guilabel:`Mapping`, 
+       :guilabel:`Code Generation`, and :guilabel:`Data Migration` views. 
+       You can also navigate back to the project home screen, 
+       configure project settings, and access the help menu.
+  
+   * - Left Pane
+     - :guilabel:`Manage` your relational connections. Filter and 
+       toggle visibility of your database entities.
+
+   * - Diagram Toolbar
+     - Select diagram layout options, add synthetic foreign keys, and 
+       create additional diagram tabs.
+
+   * - Diagrams
+     - View relational and MongoDB diagrams, view field names, and data types. 
+       Control diagram zoom level and view the entity mini map.
```

Comment:
(wording)
Maybe "map" rather than "mini map".

(Though mini map is adorable.) If you decide to keep mini-map, it should be hyphenated.

Reply:
Will opt for the adorable hyphenated option :)